pkg/utils/utils_test.go

```diff
@@ -74,8 +74,8 @@ func TestTarDir(t *testing.T) {
 	if !os.IsNotExist(err) {
 		os.Remove(tarPath)
 	}
-
-	err = TarDir(filepath.Join(testDir, "test_src"), tarPath)
+	emptyArrayOfStrings := []string{}
+	err = TarDir(filepath.Join(testDir, "test_src"), tarPath, emptyArrayOfStrings, emptyArrayOfStrings)
```

Comment:
Could you please add more test cases when the include path and exclude paths are not empty.

Reply:
updated, please check